### DISCUSSION TITLE
[Outdated] Unable to commit WebP Lossless?

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [community/community#134654] [Repositories]
"@ext/trac.ffmpeg.org/ticket/8063": ["Null-Frame-Example.webp"](https://github.com/MasterInQuestion/attach/raw/main/@ext/trac.ffmpeg.org/ticket/8063/Null-Frame-Example.webp)

    Attempted either web UI upload:
    https://github.com/MasterInQuestion/attach/upload/main/@ext/trac.ffmpeg.org/ticket/8063

    VS Code VirtualSourceControl.commitChanges:
2024-08-05 06:11:41.595 [error]
GraphqlResponseError: Request failed due to following response errors:
\- Something went wrong while executing your query. Please include &#x60;A88D:1A8B3:D31F7:E2FE1:66AFFC98&#x60; when reporting this issue.

    &#x60;git&#x60; CLI push:
Connection to ssh.github.com closed by remote host.
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
\
\
    Though already successfully pushed:
    [ https://github.com/MasterInQuestion/attach/commit/fb8ae2fc884266c89759b7a3f168c9835870e184](https://github.com/MasterInQuestion/attach/commit/fb8ae2fc884266c89759b7a3f168c9835870e184)
    .
    I'd like to learn about the exact cause.
    WebP Lossless CVE?
    Periodic IP block?
    Or else?

    Thanks.

**Comment by MasterInQuestion** [comment]
My guess is IP block.
    See also: https://github.com/orgs/community/discussions/138223

**Reply by abdulla-uoanbar** [reply]
same here, it is ip block by internet service provider, do you know how can i find which ip i can ask the ISP to open?

**Reply by MasterInQuestion** [reply]
What do you mean?
    To get your current IP: https://browserleaks.com/ip
